nimble/drivers/dialog_cmac/src/ble_phy.c

```diff
@@ -1178,6 +1186,8 @@ ble_phy_init(void)
     g_ble_phy_data.phy_whitening = 1;
 #endif
 
+    set_reg32_bits(0x40000904, 0x0004, 1);
```

Comment:
move this to ble_rf_init

Reply:
This code line deals with suppressing a meta-stability issue from raising an error condition towards the controller, mainly observed in GF variant. Preferably, this should be executed as early as possible and is not related to the RF part. Is there any specific reason why ble_rf_init() is recommended?

Reply:
in such case you can put it in `SystemInit` instead: https://github.com/apache/mynewt-core/blob/master/hw/mcu/dialog/cmac/src/system_cmac.c

Reply:
Correct. That is the place I was looking for...

Reply:
Done. https://github.com/apache/mynewt-core/pull/2992/commits/e110969a71b95f9f89578cdc5fcb2136ba9e17f3